ui/panels-plugin/src/plugins/gauge-chart/gauge-chart-model.ts

```diff
@@ -47,14 +47,14 @@ export function createInitialGaugeChartOptions(): GaugeChartOptions {
       },
     },
     calculation: 'LastNumber',
-    unit: { kind: 'Percent' },
+    unit: { kind: 'PercentDecimal' },
```

Comment:
I updated this default unit when adding a new GaugeChart panel since I've noticed that `PercentDecimal` gets used more often, but I'm open to reverting this change